integration-tests/bats/db-revision-specifiers.bats

```diff
@@ -3,6 +3,7 @@ load $BATS_TEST_DIRNAME/helper/common.bash
 
 setup() {
     setup_common
+    dolt config --add database.disablehyphen true
```

Comment:
Although... thinking about this some more... an env var has the advantage of always being "global" for a SQL server. By using a config setting, we need to check each individual database's setting (or use `dolt config --global`). There's an edge case with `undrop` where the dropped database manager uses one global setting no matter what the dropped database requested. I'm wondering if using an env var might actually be simpler. It would also mean we didn't have to pipe the boolean value around through a bunch of calls, which would be a minor advantage. We can discuss more in our call, too.